scripts/utils/ids-to-base64.py

```diff
@@ -1,30 +1,34 @@
+#!/usr/bin/env python3
+
 import base64
 import os
 import re
 import sys
 
+
 def trace_id_base64(match):
     id = int(match.group(1), 16)
     hex = '%032x' % id
     b64 = base64.b64encode(hex.decode('hex'))
     return '"traceId": "%s"' % b64
 
+
 def span_id_base64(match):
     id = int(match.group(1), 16)
     hex = '%016x' % id
     b64 = base64.b64encode(hex.decode('hex'))
-    return '"spanId": "%s"' % b64
+    return f'"spanId": "{b64}"'
+
 
 for file in sys.argv[1:]:
-    print file
-    backup = '%s.bak' % file
+    print(file)
+    backup = f'{file}.bak'
```

Comment:
I feel the named formatted strings are a bit more readable, supported in python3.